src/event.rs

```diff
@@ -262,179 +262,123 @@ pub enum Key {
+            VirtualKeyCode::Numpad3 => Key::Numpad3,
+            VirtualKeyCode::Numpad4 => Key::Numpad4,
+            VirtualKeyCode::Numpad5 => Key::Numpad5,
+            VirtualKeyCode::Numpad6 => Key::Numpad6,
+            VirtualKeyCode::Numpad7 => Key::Numpad7,
+            VirtualKeyCode::Numpad8 => Key::Numpad8,
+            VirtualKeyCode::Numpad9 => Key::Numpad9,
+
+            VirtualKeyCode::Apostrophe => Key::Apostrophe,
+            VirtualKeyCode::At => Key::At,
+            VirtualKeyCode::Backslash => Key::Backslash,
+            VirtualKeyCode::Colon => Key::Colon,
+            VirtualKeyCode::Comma => Key::Comma,
+            VirtualKeyCode::Equals => Key::Equals,
+            VirtualKeyCode::Grave => Key::Backtick,
+            VirtualKeyCode::LBracket => Key::LeftBracket,
+            VirtualKeyCode::Minus => Key::Minus,
+            VirtualKeyCode::NumpadAdd => Key::Plus,
+            VirtualKeyCode::NumpadComma => Key::NumpadComma,
+            VirtualKeyCode::NumpadDecimal => Key::Decimal,
+            VirtualKeyCode::NumpadDivide => Key::Divide,
+            VirtualKeyCode::NumpadEnter => Key::NumpadEnter,
+            VirtualKeyCode::NumpadEquals => Key::NumpadEquals,
+            VirtualKeyCode::NumpadMultiply => Key::Multiply,
+            VirtualKeyCode::NumpadSubtract => Key::Minus,
+            VirtualKeyCode::Period => Key::Period,
+            VirtualKeyCode::RBracket => Key::RightBracket,
+            VirtualKeyCode::Semicolon => Key::Semicolon,
+            VirtualKeyCode::Slash => Key::Slash,
+            VirtualKeyCode::Tab => Key::Tab,
 
             // Unsupported keys (could be changed in the future)
-            Snapshot |
-            Scroll |
-            Pause |
-            Insert |
-            Compose |
-            Caret |
-            Numlock |
-            AbntC1 |
-            AbntC2 |
-            Apps |
-            Ax |
-            Calculator |
-            Capital |
-            Convert |
-            Kana |
-            Kanji |
-            LAlt |
-            LControl |
-            LShift |
-            LWin |
-            Mail |
-            MediaSelect |
-            MediaStop |
-            Mute |
-            MyComputer |
-            NavigateForward |
-            NavigateBackward |
-            NextTrack |
-            NoConvert |
-            OEM102 |
-            PlayPause |
-            Power |
-            PrevTrack |
-            RAlt |
-            RControl |
-            RShift |
-            RWin |
-            Sleep |
-            Stop |
-            Sysrq |
-            Underline |
-            Unlabeled |
-            VolumeDown |
-            VolumeUp |
-            Wake |
-            WebBack |
-            WebFavorites |
-            WebForward |
-            WebHome |
-            WebRefresh |
-            WebSearch |
-            WebStop |
-            Yen |
-            Copy |
-            Paste |
-            Cut => return None,
+            _ => return None,
```

Comment:
Could you change this back to the way it was? The reason we didn't use a wildcard here initially was so that new variants added to `VirtualKeyCode` would cause a compilation error (because the match would not be exhaustive). This is important to make sure we know whenever that happens and can update the code accordingly.

Reply:
I don't really see what would be interesting in having to copy and paste in a huge list whenever there are new variants.

Reply:
@PaulDance It's so that we know if new keys are added that we may want to support. It also helps us in cases where an old key has a new mapping (e.g. if they eventually add a third kind of keycode for the minus symbol). Either way, it's better for us to have a compilation error than to silently miss changes in this API.